app/models/concerns/searchable.rb

```diff
@@ -85,16 +84,19 @@ def self.search(query, filter_countries, filter_cities, filter_lang)
             },
             country: {
               terms: {
-                field: 'country',
+                field: 'country.keyword',
                 size: 999
               }
             }
           }
         }
 
-      # minimum score depends completely on the given data and query, find out what works in your case.
+      # minimum score:
+      # depends completely on the given data and query, find out what works in your case.
       # can't be integrated directly in query hash because tests fail
-      query_hash[:min_score] = 0.08 if Rails.env.production?
+      # query_hash[:min_score] = 0.08 if Rails.env.production?
+      # version migration: scoring changed from 2.4.5 to 6.0, so 0.08 won't filter.
+      # if min_score stays make it higher (i.e. 3.00)
```

Comment:
I think it's good to have this comment as a PR comment, and then remove these three lines from the code. Going forward, we won't need to know the history of this in the code, and can always look at the PR for more context of the changes made.